src/config/mod.rs

```diff
@@ -1988,6 +1997,28 @@ impl Dimensions {
     }
 }
 
+/// Window Position
+#[serde(default)]
+#[derive(Default, Debug, Copy, Clone, Deserialize, PartialEq, Eq)]
+pub struct Position {
+    /// Window x-position
+    #[serde(deserialize_with = "failure_default")]
+    pub x: i32,
+
+    /// Window y-position
+    #[serde(deserialize_with = "failure_default")]
+    pub y: i32,
+}
```

Comment:
Using `Delta` does seem like a good idea.

>It might make sense to rename `Delta`

Do you mean renaming `Delta` to `Position`, or just renaming `Delta` in general?

Negative positions do indeed spawn the window outside of the screen (at least on Linux), so I'll leave that as is.

Reply:
Also, negative positions cannot be passed to the `--position` flag, but setting `AllowLeadingHyphen` fixes that. I'm assuming we want users to be able to pass negative values, so should `ArgSettings` be `use`d at the top or should I just `.set(clap::ArgSettings::AllowLeadingHyphen)`?

Reply:
> Do you mean renaming Delta to Position, or just renaming Delta in general?

I'm talking about renaming `Delta` in general. I'd say `Vector` since that's commonly used to store dimensional positions, but with `Vec` that namespace is kinda already occupied.

Since `padding` also uses a `Delta` even though `Delta` might not be the greatest name, I'm almost inclined to just keep the name though. If you have a good idea, I'm all ears, otherwise I'd just use Delta for now.

> Also, negative positions cannot be passed to the --position flag, but setting AllowLeadingHyphen fixes that. I'm assuming we want users to be able to pass negative values, so should ArgSettings be used at the top or should I just .set(clap::ArgSettings::AllowLeadingHyphen)?

Usually importing at the top of the file is a good idea, just makes things a lot cleaner in the actual code.

Does this create any issues with single-character flags though? We wouldn't want to introduce a regression and there has to be a reason why it doesn't work by default.

Reply:
I'll leave `Delta` alone for now, then.

The only issue it has from my brief testing is, if you don't specify x and y coordinates (i.e. running `cargo run -- --position -e bash`), it eats the argument after it. Without the setting, it properly errors out notifying the user that `--position` requires 2 values.

Reply:
> The only issue it has from my brief testing is, if you don't specify x and y coordinates (i.e. running cargo run -- --position -e bash), it eats the argument after it. Without the setting, it properly errors out notifying the user that --position requires 2 values.

I think we can just ignore this issue as user error. I don't think there is any well defined 'best option' to go for here.

> If we do go forward with allowing hyphen values despite that issue, I'd use .allow_hyphen_values(true), which does the same thing without needing use clap::ArgSettings

Sounds good.